src/plone/restapi/tests/test_functional_auth.py

```diff
@@ -1,4 +1,11 @@
 # -*- coding: utf-8 -*-
+import base64
```

Comment:
100% sure you already know the Plone convention, could be that a wrong import sort linter was applied here?

Reply:
Well in that case you're 100% wrong about what I already know.  ;-)  I am not aware of a Plone convention on sorting imports.  I *am* aware of some portions of the old Zope community that preferred sorted imports.  I am also aware that I do not prefer sorted imports, at least not without some division demonstrating the level in the stack, e.g.: standard library imports, framework agnostic non-standard-library imports, framework imports, imports from this add-on.

Regardless, I'm happy to follow a community convention on imports.  If this is a strong community standard, shouldn't this be in the CI checks?  Failing that, I've looked at a few other modules and it's not at all clear from those what the convention is.  So if CI isn't telling me what to change, care to give me link, @sneridagh, or otherwise help me understand what change you would prefer I make here?  Thanks!

Reply:
It is kind of best practice in the Plone community to move the "import *" statements below the other imports. We use this convention in Plone core packages and in plone.restapi. Therefore it seems indeed odd to move them to the top here.

There is/was also some initiative to automatically sort imports and make them checkable on CI, which would be indeed the way to go. Though, that never was fully enforced. Partly because it is not sufficient to just let the CI check those things, you need proper pre-commit hooks, IDE integration, etc.

That being said, you are 100% correct and this is indeed a problem for people that do not work with Plone core code on a daily basis. Though I'd suggest just not move around the "import *" to have at least a little bit of consistency in the package.

Reply:
K, thanks much for the explanation!  My default mode is to try to clean up things I touch.  I'll try to remember not to do that with imports in Plone core going forward.  I'll also revert the import order here to the way it was before.